layouts/shortcodes/sponsors_shortcode.html

```diff
@@ -0,0 +1,10 @@
+<div class="sponsor-container">
+    <div class="sponsor-image">
+      <a href="{{ index .Params 0}}">
+        <img src="{{ index .Params 1}}" width="500">
+      </a>
+    </div>
+    <p>
+        {{ index .Params 2}}<strong> {{ index .Params 3}}</strong> {{ index .Params 4}}
```

Comment:
Instead of 3 params, you can use 1 which can be supplied markdown (where content writer will do bolding as per their preference)